CHANGES.md

```diff
@@ -19,6 +19,15 @@
 * `xcube serve` now also serves datasets that are located in 
   subdirectories of filesystem-based data stores such as
   "file", "s3", "memory". (#579)
+
+* `xcube serve` no longer rejects datasets whose spatial
+  resolutions in x any differ less than 1%. (#590)
+  It also no longer rejects datasets with large dimension 
+  sizes. (Formerly, an integer-overflow occurred in size 
+  computation.) 
+
+* `DatasetChunkCacheSize` is now optional in `xcube serve`
+  configuration. (Formerly, when omitted, the server crashed.)
```

Comment:
I don't see this in this PR.

Reply:
xcube/webapi/context.py, line 479, see also changes in files

Reply:
Right, I see.